package.json

```diff
@@ -48,7 +48,7 @@
     "react-native-get-random-values": "^1.5.1",
     "react-native-inappbrowser-reborn": "^3.5.1",
     "react-native-localize": "^1.4.3",
-    "react-native-material-menu": "^1.2.0",
+    "react-native-material-menu": "git+https://github.com/ankitecd/react-native-material-menu.git",
```

Comment:
Can you describe why a fork was necessary?

Reply:
ScrollView was not present in the package. So I added one as it was not allowing to make the menu options fixed size so that scroll can be implemented for large list.